src/modules/products/schemas/create-product.schema.ts

```diff
@@ -0,0 +1,67 @@
+import { z } from "zod";
+
+export const CreateProductSchema = z.object({
+  name: z
+    .string({
+      required_error: "",
+    })
+    .max(30, {
+      message: "Name must be less than 30 characters.",
+    }),
+  description: z
+    .string({
+      required_error: "",
+    })
+    .max(80, {
+      message: "Description must be less than 80 characters.",
+    }),
```

Comment:
Podríamos agregar los mínimos

```suggestion
  name: z
    .string({
      required_error: "",
    })
    .min(3, {
      message: "Name must be at least 3 characters",
    })
    .max(30, {
      message: "Name must be less than 30 characters.",
    }),
  description: z
    .string({
      required_error: "",
    })
    .min(5, {
      message: "Description must be at least 5 characters",
    })
    .max(80, {
      message: "Description must be less than 80 characters.",
    }),
```

Reply:
Tenes razón! Lo olvidé